test/functional/terminal/tui_spec.lua

```diff
@@ -2703,8 +2725,8 @@ describe("TUI as a client", function()
     local client_super = spawn_argv(true)
 
     set_session(server)
-    local server_pipe = eval'v:servername'
-    feed'iHalloj!<esc>'
```

Comment:
this doesn't need to change. `set_session(server)` makes global calls, including `feed`, in the context of the server.

Reply:
I just want to avoid using `feed()` in this test file, because it easy to mix it up with `feed_data()`, causing data races.